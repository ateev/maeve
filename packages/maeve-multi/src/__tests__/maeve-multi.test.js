import React from 'react';
import { shallow } from 'enzyme';
import MaeveMulti from '../maeve-multi.jsx';

test('render maeve multi', () => {
  const component = shallow(
    <MaeveMulti />
  );
  expect(component).toMatchSnapshot();
});