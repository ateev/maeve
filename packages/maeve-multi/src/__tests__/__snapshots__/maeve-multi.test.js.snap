exports[`test render maeve multi 1`] = `
<div
  className="maeve-multi">
  <div
    className="maeve-multi-item">
    Hi
    <div
      className="remove-button"
      onClick={[Function]}>
       - 
    </div>
  </div>
  <div
    className="add-button"
    onClick={[Function]}>
     + 
  </div>
</div>
`;
